operators/rename_from_scene_strip.py

```diff
@@ -0,0 +1,49 @@
+import bpy
+import operator
+
+from bpy.props import StringProperty
+from .utils.doc import doc_name, doc_idname, doc_brief, doc_description
+
+class RenameStripScene(bpy.types.Operator):
+    """
+    Rename a Scene Strip and its source scene.
+    """
+    doc = {
+        'name': doc_name(__qualname__),
+        'demo': '',
+        'description': doc_description(__doc__),
+        'shortcuts': [],
+        'keymap': 'Sequencer'
+    }
+
+    bl_idname = doc_idname(doc['name'])
+    bl_label = doc['name']
+    bl_description = doc_brief(doc['description'])
+    bl_options = {"REGISTER", "UNDO"}
+
+    new_name = StringProperty(
+        name="Strip New Name",
+        description="The name both the SceneStrip and its source Scene will take",
+        default="")
+
+    @classmethod
+    def poll(cls, context):
+        if context.selected_sequences == None or len(context.selected_sequences) < 1:
+            return False
+
```

Comment:
I think this should do it, right @razcore-art ?

Reply:
yes this works. You should really use `context.selected_sequences is None` tho' :p